src/ontology/cl-edit.owl

```diff
@@ -27782,32 +27792,15 @@ SubClassOf(obo:CL_4023050 obo:CL_4023040)
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001384))
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
 
-# Class: obo:CL_4023048 (L4/5 intratelencephalic projecting glutamatergic neuron of the primary motor cortex (Mus musculus))
-
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33184512") obo:IAO_0000115 obo:CL_4023048 "An intratelencephalic-projecting glutamatergic with a soma located in upper L5 of the primary motor cortex. These cells have thin untufted apical dendrites.")
-AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023048 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023048 "ILX:0770174")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023048 "L4/5 IT MOp (Mus)")
-AnnotationAssertion(oboInOwl:inSubset obo:CL_4023048 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
-AnnotationAssertion(rdfs:label obo:CL_4023048 "L4/5 intratelencephalic projecting glutamatergic neuron of the primary motor cortex (Mus musculus)")
-SubClassOf(obo:CL_4023048 obo:CL_4023040)
-SubClassOf(obo:CL_4023048 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001384))
-SubClassOf(obo:CL_4023048 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
-
-# Class: obo:CL_4023047 (L2/3 intratelencephalic projecting glutamatergic neuron of the primary motor cortex (Mus musculus))
-
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023047 "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 2/3.")
-AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023047 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023047 "L2/3 IT MOp (Mus)")
-AnnotationAssertion(oboInOwl:inSubset obo:CL_4023047 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
-AnnotationAssertion(rdfs:label obo:CL_4023047 "L2/3 intratelencephalic projecting glutamatergic neuron of the primary motor cortex (Mus musculus)")
-SubClassOf(obo:CL_4023047 obo:CL_4023040)
-SubClassOf(obo:CL_4023047 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001384))
-SubClassOf(obo:CL_4023047 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
-
-
-
+# Class: obo:CL_4023051 (L4 intratelencephalic projecting glutamatergic neuron of the primary motor cortex (Mus musculus))
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33184512") obo:IAO_0000115 obo:CL_4023051 "An intratelencephalic-projecting glutamatergic neuron that is located in the boundary between L2/3 and L5 of the primary motor cortex. These cells have diverse morphologies with some pyramidal and some stellate cells.")
```

Comment:
This strikes me as a heterogenous grouping class that only makes sense in the context of single cell transcriptomics clustering.  I wonder whether we should make that explicit + define using markers (RF markers could work well for this).  I think worth holding off for further discussion.